melanges-de-linstitut-dominicain-detudes-orientales-en.csl

```diff
@@ -0,0 +1,260 @@
+        <choose>
+          <if type="chapter" match="any">
+            <text macro="author"/>
+            <text macro="title"/>
+            <group delimiter=" ">
+              <text term="in" font-style="normal"/>
+              <choose>
+                <if match="any" variable="editor">
+                  <group delimiter=", ">
+                    <text macro="editor"/>
+                    <text macro="container-title"/>
+                  </group>
+                </if>
+                <else>
+                  <text macro="container-title"/>
+                </else>
+              </choose>
+            </group>
+            <text macro="publisher-place"/>
+            <text macro="publisher"/>
+            <choose>
+              <if match="any" variable="original-date">
+                <group delimiter=" ">
+                  <text macro="issued"/>
+                  <text macro="original-date" prefix="[" suffix="]"/>
+                </group>
+              </if>
+              <else>
+                <text macro="issued"/>
+              </else>
+            </choose>
+            <text macro="page"/>
+          </if>
+          <else-if type="article article-journal article-magazine article-newspaper" match="any">
+            <text macro="author"/>
+            <text macro="title"/>
+            <group delimiter=" ">
+              <text macro="container-title"/>
+              <text macro="volume"/>
+            </group>
+            <text macro="issue"/>
+            <text macro="issued"/>
+            <text macro="page"/>
+          </else-if>
+          <else-if type="entry-encyclopedia entry-dictionary" match="any">
+            <text macro="author"/>
+            <text macro="title"/>
+            <choose>
+              <if match="any" variable="editor">
+                <group delimiter=" ">
+                  <text term="in"/>
+                  <text macro="editor"/>
+                </group>
+              </if>
+            </choose>
+            <choose>
+              <if match="any" variable="volume">
+                <text macro="container-title"/>
+                <group delimiter=" ">
+                  <label plural="never" variable="volume" form="short"/>
+                  <number variable="volume"/>
+                  <group delimiter=", " prefix="(" suffix=")">
+                    <text macro="edition"/>
+                    <text macro="issued"/>
+                  </group>
+                </group>
+              </if>
+              <else>
+                <group delimiter=" ">
+                  <text macro="container-title"/>
+                  <group delimiter=", " prefix="(" suffix=")">
+                    <text macro="edition"/>
+                    <text macro="issued"/>
+                  </group>
+                </group>
+              </else>
+            </choose>
+            <text macro="page"/>
+          </else-if>
+          <else-if type="webpage" match="any">
```

Comment:
```suggestion
          <else-if type="webpage article" match="any">
```

Reply:
I prefer to leave the treatment of the article category in the else instruction, which applies to all categories not described above